### ISSUE TITLE
I knew it！

### ISSUE THREAD

**Issue by CiaoHe** [lucidrains/alphafold3-pytorch#2]
Come on, keep going forward！

**Comment by CiaoHe** [comment]
Feel like backing to the days of AF2. Unfortunately, DeepMind chose API this time.

**Comment by lucidrains** [comment]
@CiaoHe indeed, although i feel like this could be a more incremental paper than AF2

should still get the code out there though for reproduction and continued research.

**Comment by CiaoHe** [comment]
But two points are highlighted: fully embrace the diffusion model and rationally use MSA information (fewer modules to process MSA)

I am curious about how they input the information of ligand and RNA into the encoder and reconstruct them synchronously with protein structure during decoding(or say, generation)

**Comment by lucidrains** [comment]
@CiaoHe ah i see

yea AF2 is basically an iterative refinement system with the recycling, so going full diffusion makes a lot of sense

**Comment by CiaoHe** [comment]
> @CiaoHe ah i see
> 
> 
> 
> yea AF2 is basically an iterative refinement system with the recycling, so going full diffusion makes a lot of sense
> 
> 
> 
> edit: they involve RNA sequences? is this something new?

I saw they report several tasks on  "Nucleic Acids" like Protein-RNA and Protein-dsDNA (in the last page).

**Comment by lucidrains** [comment]
@CiaoHe oh got it, for predicting protein-nucleic acid complexes, makes sense

yea i'll read the paper soon, after i wrap up another project

**Comment by celalp** [comment]
They do include nucleic acids but they are rather short and according to the paper are more prone to hallucinations and weird clashing atoms. They also say that they used 256 A100s for 10 days, there is also that.

**Comment by everyday847** [comment]
Nucleic acids aren't especially short; the large ribosomal subunit of E coli is around 3000 nt. I'd say the modal size of an RNA domain is smaller (in number of monomers) than a protein domain, but it's comparable in number of dofs.
